testutils/resolver.go

```diff
@@ -0,0 +1,85 @@
+package testutils
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/renproject/darknode/jsonrpc"
+	"github.com/renproject/darknode/tx"
+	"github.com/renproject/phi"
+	"github.com/sirupsen/logrus"
+)
+
+type MockResolver struct {
```

Comment:
Thought it might be useful to hook into; but the mock watcher (testify) that I tried was causing races, so pulled it out. Committed a version using the existing mock.